EventView-1/src/main/java/com/eventview/controller/EventTypesRestController.java

```diff
@@ -22,78 +25,62 @@
 
     @GetMapping(path = "/types")
```

Comment:
why 'types' here n 'type' in other place

Reply:
fixed